core/modules/layout/includes/block.hero.inc

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ *  BlockHero extends BlockText
+ *
+ *  This class allows us to create hero blocks with image backgrounds.
+ */
+class BlockHero extends BlockText {
+  /**
+   * {@inheritdoc}
+   */
+  function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+
+    $this->settings += array(
+      'image' => '',
+      'image_path' => '',
+    );
+  }
+
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Hero block');
+  }
+
+  /**
+   *  Builds the block's configuration form.
+   */
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    $upload_validators = array(
+      'file_validate_extensions' => array('jpg jpeg png gif'),
+      'file_validate_image_resolution' => array('3200x1600', '1200x600'),
```

Comment:
What if min-height was in vh's. I think that's probably was most important here, don't be taller than the screen, and better yet, be a little shorter than the screen (`90vh`?)